cogs/github.py

```diff
@@ -0,0 +1,101 @@
+from discord import Embed, Color
+from discord.ext.commands import Cog, Context, command, Bot, cooldown, BucketType
+from .utils.constants import Emojis
+import aiohttp
+
+BAD_RESPONSES = {
+    404: "Issue/pull request not Found! Please enter a valid PR Number!",
+    403: "Rate limit is hit! Please try again later!"
+}
+
+
+# TODO : add gitlab commands too ig.
+class Github(Cog):
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+        self.session = aiohttp.ClientSession()
+
+    @classmethod
+    def generate_description(cls, description: str, stars: str, forks: str, command: str) -> str:
+        return f"*{description}*\nStars: {stars} Forks: {forks}\n Command: {command}"
```

Comment:
Why a classmeth ? Make that a static method
```suggestion
    @staticmethod
    def generate_description(description: str, stars: str, forks: str, command: str) -> str:
        return f"*{description}*\nStars: {stars} Forks: {forks}\n Command: {command}"
```